docs/docs/scheme-intro/data-types.md

```diff
@@ -59,7 +59,7 @@ Escape character used by Terminal emulators to add formatting (like colors).
 
 ## Characters
 
-You can define single character as data type
+Single characters can be defined as a distinct data type.
```

Comment:
I prefer:

> You can define a single character as a data type

I don't see a reason to change the structure of the sentence and swap stuff inside.